### DISCUSSION TITLE
Welcome to @giscus/react component!

### DISCUSSION THREAD

**Discussion by giscus** [Cheolwan-Park/next-notion-blog#26] [Announcements]
# Welcome to @giscus/react component!

Notion을 CMS로 이용하는 블로그 개발기.
Next.js, Notion API를 활용해서 블로그를 만들어 봤습니다.

http://codeserver.codingvillain.com:8000/post/notion-blog-dev-review

**Comment by cheolwanpark** [comment]
테스트!!!
